testing/qgb.go

```diff
@@ -13,3 +21,19 @@ func NewEVMClient(key *ecdsa.PrivateKey) *evm.Client {
 	// the simulated backend doesn't provide an RPC endpoint.
 	return evm.NewClient(logger, nil, key, "", 100000000)
 }
+
+func NewOrchestrator(
+	node *TestNode,
+) *orchestrator.Orchestrator {
+	logger := tmlog.NewNopLogger()
+	appQuerier := rpc.NewAppQuerier(logger, node.CelestiaNetwork.GRPCClient, encoding.MakeConfig(app.ModuleEncodingRegisters...))
+	tmQuerier := rpc.NewTmQuerier(node.CelestiaNetwork.Client, logger)
+	p2pQuerier := p2p.NewQuerier(node.DHTNetwork.DHTs[0], logger)
+	broadcaster := orchestrator.NewBroadcaster(node.DHTNetwork.DHTs[0])
+	retrier := orchestrator.NewRetrier(logger, 3, 500*time.Millisecond)
+	orch, err := orchestrator.New(logger, appQuerier, tmQuerier, p2pQuerier, broadcaster, retrier, *celestiatestnode.NodeEVMPrivateKey)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
[optional]
why not pass a testing.T as an arg to handle errors? are we using this function outside of tests?

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/153